### ISSUE TITLE
jQuery $ in variable names

### ISSUE THREAD

**Issue by joshcummingsdesign** [WordPress/twentyseventeen#277]
As a style, I would think that the dollar sign in javascript files would only be used for variables that store jQuery objects, not values.

For example you would write:

``` javascript
var $navigation = $( '.navigation-top' );
```

But not:

``` javascript
$navigationHeight = $navigation.outerHeight();
```

The former is returning a collection of matched elements, and the latter is returning a float.

Therefore it can be rewritten like so:

``` javascript
navigationHeight = $navigation.outerHeight();
```

A small detail and maybe a nice-to-have, but thought it would be good to discuss.

**Comment by davidakennedy** [comment]
Thanks @joshcummingsdesign for the issue. It would be good to look at Twenty Sixteen and Twenty Fifteen for guidance here as this issue sounds familiar to me.

**Comment by joshcummingsdesign** [comment]
#306 Is an example of global.js refactored for optimization and addresses the jQuery '$' sign issue.

**Comment by joshcummingsdesign** [comment]
Interestingly enough, $ is used inconsistently in Twenty Sixteen. It does, however, use the IIFE pattern like so:

``` javascript
( function( $ ) { ... }( jQuery ) )
```

**Comment by tywayne** [comment]
Relevant discussion from twentysixteen - https://github.com/WordPress/twentysixteen/issues/214
It was decided back then to not use the $ prefix at all, but the slack discussion linked in that issue shows that decision was mainly based on it not being an official guideline.

FWIW, I prefer using it to denote jQuery objects, like the PR here does. Just noting the historical discussion for context.

**Comment by laurelfulford** [comment]
Merged in #306 & closing - thanks @joshcummingsdesign!
